rs/nns/governance/src/storage.rs

```diff
@@ -26,6 +27,9 @@ const NEURON_FOLLOWING_INDEX_MEMORY_ID: MemoryId = MemoryId::new(11);
 const NEURON_KNOWN_NEURON_INDEX_MEMORY_ID: MemoryId = MemoryId::new(12);
 const NEURON_ACCOUNT_ID_INDEX_MEMORY_ID: MemoryId = MemoryId::new(13);
 
+const NP_REWARDS_LOG_INDEX_MEMORY_ID: MemoryId = MemoryId::new(14);
```

Comment:
NCR. I'm not always against acronyms, but in this specific case I don't feel NP is a significant enough concept.